alf/utils/math_ops.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gin
+import tensorflow as tf
+"""Various math ops."""
+
+
+@gin.configurable
+def clipped_exp(value, clip_value_min=-20, clip_value_max=2):
```

Comment:
same implementation with `alf.utils.common.std_clip_transform` , should remove one of them